cfn.template.yaml

```diff
@@ -29,6 +35,18 @@ Mappings:
       Value: api
 
 Resources:
+  ALBRecordSet:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      AliasTarget:
+        DNSName: !GetAtt ConsentLoadBalancer.DNSName
+        HostedZoneId: !GetAtt ConsentLoadBalancer.CanonicalHostedZoneID
+      HostedZoneId: !Ref HostedZoneId
+      Name: !Ref DomainName
+      Region: !Ref AWS::Region
```

Comment:
I don't think this or the SetIdentifier are required? The docs say "Latency-based resource record sets only" for the former and "Resource record sets that have a routing policy other than simple" for the latter?

Reply:
Not sure about Region; I left it there as it's available to the template.

That's what I thought about the SetIentifier too, however, the SetIdentifier is required when creating it via CloudFormation. The UI however allows you to create it without a SetIdentifier 🙃